MdeModulePkg/Core/PiSmmCore/PiSmmIpl.c

```diff
@@ -525,7 +525,7 @@ SmmBase2GetSmstLocation (
   after SetVirtualAddressMap().
 
   @param[in] This                The EFI_SMM_COMMUNICATION_PROTOCOL instance.
-  @param[in, out] CommBuffer     A pointer to the buffer to convey into SMRAM.
+  @param[in, out] CommBuffer     A pointer to the virtual address of the buffer to convey into SMRAM.

```

Comment:
This change should be removed to match original.  CommBuffer has to be physical.

Reply:
Right, forgot to push the last change. Should be updated now.